lunatrace/bsl/hasura/metadata/databases/lunatrace/tables/npm_revision.yaml

```diff
@@ -0,0 +1,34 @@
+table:
```

Comment:
If hasura isn't being used any more we should probably remove this so it's not exposed in gql schema